server/src/main/java/org/apache/druid/segment/loading/StorageLocation.java

```diff
@@ -44,19 +45,25 @@
   private static final Logger log = new Logger(StorageLocation.class);
 
   private final File path;
-  private final long maxSize; // in bytes
+  private final long maxSizeBytes;
   private final long freeSpaceToKeep;
 
-  // Set of files stored under the given path. All accesses must be synchronized with currSize.
+  /**
+   * Set of files stored under the given path. All accesses must be synchronized with currSizeBytes.
```

Comment:
Same about "All accesses must be synchronized with currSizeBytes."

Reply:
Could be `under {@link #path}.`

Reply:
Done.